etc/config.xml

```diff
@@ -17,6 +17,16 @@
                 <can_capture>1</can_capture>
                 <payment_action>authorize</payment_action>
             </omise_cc>
+
+            <omise_offsite_internetbanking>
+                <active>0</active>
+                <title>Internet Banking</title>
+                <model>OmiseAdapter</model>
```

Comment:
Note, this model will be changed in the next PR. (to `OmiseOffsiteInternetbankingAdapter` model instead).